src/mame/sinclair/sprinter.cpp

```diff
@@ -0,0 +1,1758 @@
+// license:BSD-3-Clause
+// copyright-holders:Andrei I. Holub
+/*******************************************************************************************
+
+Sprinter Sp2000 (Peters Plus Ltd)
+
+Hardware:
+- CPU                   Z84C15 (21MHz/3.5MHz)
+- RAM                   4Mb (64Mb)
+- Fast RAM              64Kb
+- ROM                   256Kb
+- Video RAM             256Kb (512Kb)
+- FDD controller        WD1793
+- Support FDD:          3,5" disk (1.44Mb/720Kb) / 5,25" disk (720Kb)
+- CMOS                  DALLAS
+- HDD controller        IDE/AT
+- Keyboard controler    101key/AT
+- Mouse controller      MS-Mouse
+- Slots                 ISA-8
+- Audio out             AY-3-8910 (in PLD), Stereo 8 bit (16 bit)
+- Video out             TV, CGA analog monitor, RGB
+- Graphic mode          320x256x256, 640x256x16, Spectrum standard screen
+- Text mode             80x32x16
+
+Refs:
+	https://web.archive.org/web/20030208004427/http://www.petersplus.com/sprinter/
+
+TODO:
+- ISA memory slots
+- fully untied from Spectrum parent
+- better rendering (currently not fully discovered) in Game Configuration
+- ? detect loading Configuration by checksum, not by presents in fastram
+
+*******************************************************************************************/
+
+#include "emu.h"
+
+#include "beta_m.h"
+#include "spec128.h"
+#include "speaker.h"
+#include "tilemap.h"
+
+#include "bus/ata/atapicdr.h"
+#include "bus/ata/ataintf.h"
+#include "bus/ata/idehd.h"
+#include "bus/isa/isa.h"
+#include "bus/isa/isa_cards.h"
+#include "bus/pc_kbd/keyboards.h"
+#include "bus/pc_kbd/pc_kbdc.h"
+#include "bus/rs232/hlemouse.h"
+#include "bus/rs232/rs232.h"
+#include "bus/spectrum/zxbus.h"
+#include "cpu/z80/z84c015.h"
+#include "machine/ds128x.h"
+#include "sound/ay8910.h"
+#include "sound/dac.h"
```

Comment:
Please move the two headers from src/emu (speaker.h and tilemap.h) below the headers from src/devices.  The order of the rest of the headers is fine.